dev/sg/linters/go_checks.go

```diff
@@ -1,9 +1,42 @@
 package linters
 
-import "github.com/sourcegraph/sourcegraph/dev/sg/internal/lint"
+import (
+	"context"
+	"errors"
+	"strings"
+
+	"github.com/sourcegraph/sourcegraph/dev/sg/internal/lint"
+	"github.com/sourcegraph/sourcegraph/dev/sg/internal/repo"
+)
 
 var (
 	goFmt          = lint.RunScript("Go format", "dev/check/gofmt.sh")
 	goLint         = lint.RunScript("Go lint", "dev/check/go-lint.sh")
 	goDBConnImport = lint.RunScript("Go pkg/database/dbconn", "dev/check/go-dbconn-import.sh")
 )
+
+func lintSGExit() lint.Runner {
+	const header = "Lint dev/sg exit signals"
+
+	return func(ctx context.Context, s *repo.State) *lint.Report {
+		diff, err := s.GetDiff("dev/sg/***.go")
```

Comment:
Shouldn't it be this? Might be wrong though, maybe this is the new way of doing "give me all the things"-globs 😄 

```suggestion
		diff, err := s.GetDiff("dev/sg/**/*.go")
```

Reply:
Crazy, but I actually wasn't able to get `**/*.go` to give me everything - `dev/sg/main.go` would refuse to show up 😬